src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -3,64 +3,107 @@
 import DetailDisplay from '../../../components/DetailDisplay';
 import WorkPackageStageChip from '../../../components/WorkPackageStageChip';
 import { timelinePipe } from '../../../utils/pipes';
+import { Construction, Work } from '@mui/icons-material';
+import ScheduleIcon from '@mui/icons-material/Schedule';
+import StackedLineChartIcon from '@mui/icons-material/StackedLineChart';
+import DoneAllIcon from '@mui/icons-material/DoneAll';
+import { Link as RouterLink } from 'react-router-dom';
+import { routes } from '../../../utils/routes';
 
 interface WorkPackageDetailsProps {
   workPackage: WorkPackage;
+  dependencies: WorkPackage[];
 }
 
-const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage }) => {
+const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage, dependencies }) => {
   return (
-    <PageBlock
-      title={'Work Package Details'}
-      headerRight={
-        <Box>
-          {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
-          <WbsStatus status={workPackage.status} />
-        </Box>
-      }
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Lead" content={fullNamePipe(workPackage.projectLead)} paddingRight={2} />
+    <>
+      <Grid container columnSpacing={2} justifyContent="flex-start" sx={{ mb: 2, mt: 1 }}>
+        <Grid item display="flex" alignItems="center">
+          <Typography variant="h5">Work Package Details</Typography>
         </Grid>
+        <Grid item display="flex" alignItems="center">
+          <Box>{workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}</Box>
+        </Grid>
+
+        <Grid item display="flex" alignItems="center">
+          <Box>
+            <WbsStatus status={workPackage.status} />
+          </Box>
+        </Grid>
+      </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Start Date" content={datePipe(workPackage.startDate)} paddingRight={2} />
+      <Grid container spacing={2}>
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <Construction sx={{ mr: 2 }} />
+          <DetailDisplay label="Lead" content={fullNamePipe(workPackage.projectLead)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Progress" content={percentPipe(workPackage.progress)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <ScheduleIcon sx={{ mr: 2 }} />
+          <DetailDisplay label="Start Date" content={datePipe(workPackage.startDate)} paddingRight={1} />
         </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Manager" content={fullNamePipe(workPackage.projectManager)} paddingRight={2} />
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
```

Comment:
the grids with the flexes i think we should just get rid of, making these a chip would totally get rid of the need for them but u could also make them inline idk